src/nvim/api/window.c

```diff
@@ -57,7 +58,6 @@ void nvim_win_set_buf(Window window, Buffer buffer, Error *err)
                   window);
   }
 
-  try_start();
   int result = do_buffer(DOBUF_GOTO, DOBUF_FIRST, FORWARD, buf->b_fnum, 0);
   if (!try_end(err) && result == FAIL) {
```

Comment:
`try_start` was removed. Is the `try_end` paired with some other `try_start` ? We should try to keep them at the same "level" ..

Reply:
no, it was just a mistake.